samples/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Use the official Node.js 10 image.
+# https://hub.docker.com/_/node
+FROM node:10
+
+# Create and change to the app directory.
+WORKDIR /usr/src/app
+
+# Copy application dependency manifests to the container image.
+# A wildcard is used to ensure both package.json AND package-lock.json are copied.
+# Copying this separately prevents re-running npm install on every code change.
+COPY package.json package*.json ./
+
+# Install production dependencies.
+RUN npm install --only=production
+
+# Copy local code to the container image.
+COPY . .
+
+# Run the web service on container startup.
+CMD [ "npm", "start" ]
```

Comment:
Well hello!  This is kinda new :) I'm assuming this is to better support cloud run?  Are we planning on dropping a Dockerfile in the root of all the sample repos?

Reply:
Yes, it is. The docs associated with this will show how to used authenticated tasks with Cloud Run.